cmd/manager.go

```diff
@@ -337,6 +353,17 @@ func runManager(
 		return fmt.Errorf("unable to create controller: %w", err)
 	}
 
+	// Setup Loops contact controller
+	if err := (&controller.LoopsContactController{
+		Client:                   mgr.GetClient(),
+		Loops:                    *loopsEmailProvider,
```

Comment:
Small nit but not a blocker: pass in a config file with these fields instead of each field independently